akka-cluster/src/test/scala/akka/cluster/MetricsCollectorSpec.scala

```diff
@@ -5,6 +5,8 @@
 
 package akka.cluster
 
+// TODO remove metrics
+
 import scala.language.postfixOps
```

Comment:
Since the tests are already copied over to the new module, do we need these duplicates?

See copy: https://github.com/akka/akka/pull/16475/files#diff-5046828aa5283d40af0f5a023ef45343R21